lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1704,30 +1710,34 @@ export class UserAgentApplication {
    */
   /* tslint:disable:no-string-literal */
   //TODO: Break this function - too long
-  private saveAccessToken(authority: string, tokenResponse: TokenResponse, user: User, clientInfo: string, idToken: IdToken): void {
+  private saveAccessToken(response: AuthResponse, authority: string, parameters: any, clientInfo: string): AuthResponse {
     let scope: string;
+    let newResponse = response;
```

Comment:
rename i/p parameter to may be responseInProgress and this to 'response'

Reply:
This is internal use only.